README.md

```diff
@@ -6,6 +6,18 @@ into a human-readable XLSX format.
 These files are subsequently emailed to various stakeholders as part of a service run
 by the Online Trade Tariff team.
 
+The following is a flowchart of the process
+
+```mermaid
+graph TD
+    A[Circle CI Workflow at 7 am UTC] --> B[Sync files from trade-tariff-reporting S3 bucket]
+    B --> C[Download new files from SDES]
```

Comment:
It's probably better to expand the Acronyim,
Secure Data Exchange Service (SDES)